CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -559,7 +559,7 @@ local function CreateSettingsHub()
 			--to update, or we can just avoid using it so soon.
 			RunService.Heartbeat:wait()
 
-			if shouldShowBottomBar() then
+			if this.Visible and shouldShowBottomBar() then
```

Comment:
This callback always runs. :( I was going to submit another PR shortly after this that refactors it so that it runs only while the menu is visible and runs it once when the menu becomes visible.